### ISSUE TITLE
Clearer start button and colour contrast on the campaign pages (light mode)

### ISSUE THREAD

**Issue by Neonckb** [Alive24/CKBoost#32]
Environment: Windows 11 desktop, Edge browser

On the campaign overview page the button to proceed could be clearer, especially on light mode. The colour contrast between button and bg isn't very noticeable so it kind of looks just like text there. On dark mode it's more apparent. If the colour contrast generally could be improved (maybe the start buttons should have a solid or gradient colour bg), and the "view all quests" button is brought central and maybe the wording is changed to "Get Started!" (which then links to the quest page), I think it will make the flow clearer.

<img width="1365" height="596" alt="Image" src="https://github.com/user-attachments/assets/15a02dd1-4bf3-4e8e-be57-f8f1f55e73ad" />

**Comment by Alive24** [comment]
Should have been fixed in 28b6dc5c025d2ea6f04665ac7c7a42b4c49367d8
